native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m

```diff
@@ -20,13 +20,13 @@ @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     // Initialize default options for joining conferences.
-    MeetHourConferenceOptions *defaultOptions
-        = [MeetHourConferenceOptions fromBuilder:^(MeetHourConferenceOptionsBuilder *builder) {
-            builder.serverURL = [NSURL URLWithString:@"https://meethour.io"];
-            builder.welcomePageEnabled = NO;
-            [builder setFeatureFlag:@"ios.recording.enabled" withBoolean:YES];
-        }];
-    [MeetHour sharedInstance].defaultConferenceOptions = defaultOptions;
+//    MeetHourConferenceOptions *defaultOptions
+//        = [MeetHourConferenceOptions fromBuilder:^(MeetHourConferenceOptionsBuilder *builder) {
+//            builder.serverURL = [NSURL URLWithString:@"https://meethour.io"];
+//            builder.welcomePageEnabled = NO;
+//            [builder setFeatureFlag:@"ios.recording.enabled" withBoolean:YES];
+//        }];
+//    [MeetHour sharedInstance].defaultConferenceOptions = defaultOptions;
```

Comment:
@chepury-bharathraj  - Commented these lines.

Reply:
Not Required in appDelegate.

Reply:
Let's remove those comments.

Reply:
Removed.